pkg/controller/stacks/hosted/config_test.go

```diff
@@ -189,7 +189,7 @@ func TestGetHostPort(t *testing.T) {
 				urlHost: string(0x7f),
 			},
 			want: want{
-				err: errors.Wrap(errors.New("parse \u007f: net/url: invalid control character in URL"), "cannot parse URL"),
+				err: errors.Wrap(errors.New(`parse "\u007f": net/url: invalid control character in URL`), "cannot parse URL"),
```

Comment:
Seems like these tests are failing because we're testing through to the stdlib, which has seemingly changed its quoting behaviours in 1.14. We could try using `%q` (quoted value) rather than `%s` or `%v` in our error format strings and see if that fixes it. If not, we could use an alternative to the `test.EquateErrors` comparison option for these tests that matches less strictly.

https://godoc.org/github.com/crossplane/crossplane-runtime/pkg/test#EquateErrors
https://pkg.go.dev/github.com/google/go-cmp/cmp?tab=doc#Option

Reply:
I solved this by calling url.Parse() in the tests too, so the error message will always match no matter which golang version is used.